platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -1001,6 +1003,26 @@ public boolean requestPermission(String p_name) {
 		return true;
 	}
 
+	public void requestPermissions() {
```

Comment:
This should return a `boolean` to notify the caller whether the request was done.

Reply:
I have added return `true`.